src/status_im/contexts/chat/events.cljs

```diff
@@ -200,9 +200,10 @@
 (rf/defn navigate-to-chat
   "Takes coeffects map and chat-id, returns effects necessary for navigation and preloading data"
   {:events [:chat/navigate-to-chat]}
-  [{db :db :as cofx} chat-id animation]
+  [{db :db :as cofx} chat-id animation community-id]
   (rf/merge cofx
-            {:dispatch [(if animation :shell/navigate-to :navigate-to) :chat chat-id animation]}
+            {:dispatch [(if animation :shell/navigate-to [:navigate-to {:community-id community-id}]) 
```

Comment:
not directly related to PR, but I think we can also remove `:shell/navigate-to` here